sdk/src/main/java/io/opentelemetry/sdk/trace/SpanBuilderSdk.java

```diff
@@ -170,6 +171,7 @@ public Span startSpan() {
         spanName,
         spanKind,
         parentContext != null ? parentContext.getSpanId() : null,
+        parentContext != null ? parentContext.isRemote() : null,
```

Comment:
Consider to use https://docs.oracle.com/javase/7/docs/api/java/lang/Boolean.html#valueOf(boolean)